<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>433</width>
    <height>205</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget{
background-color: rgb(230,230,230);

}

QPushButton {
	background-color: rgb(255,255,255);  /* Background color */
    color: rgb(0, 0, 150);  /* Text color */
    border: 2px solid rgb(255,155, 50);  /* Border color */
    border-radius: 5px;  /* Rounded edges */
	padding: 2px;
}

QPushButton:hover {
    background-color: rgb(245, 245, 255);  /* Background color when hovered */
	color: rgb(0, 0, 200);
    border-color:rgb(225, 105, 0);  /* Border color when hovered */
}

QPushButton:pressed {
   background-color: rgb(200,200, 210);  /* Background color when pressed */
	border: 3px solid rgb(205, 85, 0);
}
QPushButton:clicked {
   background-color: rgb(200, 200, 210);  /* Background color when pressed */
	border:  rgb(205, 85, 0);
}

QPushButton:checked {
    background-color: rgb(220,120, 15);
	border: 3px solid rgb(205, 105, 0);
	font-weight: bold;  /* Bold text */
}

QPushButton:disabled {
    background-color: rgb(220,220, 220);
	border: 3px solid rgb(205, 105, 0);
	font-weight: bold;  /* Bold text */
}

QDoubleSpinBox:enabled{
background-color: rgb(255,255,255);
color: rgb(0, 0, 150);  /* Text color */
border: 1px solid rgb(255,255,255);  /* Border color */
border-radius: 1px;  /* Rounded edges */
padding : 2px;
}
QDoubleSpinBox:disabled{
color: rgb(0, 0, 200);  /* Text color */
border: 1px solid rgb(255,255,255);  /* Border color */
border-radius: 1px;  /* Rounded edges */
padding : 2px;
}

QSpinBox:enabled{
background-color: rgb(255,255,255);
color: rgb(0, 0, 150);  /* Text color */
border: 1px solid rgb(255,255,255);  /* Border color */
border-radius: 1px;  /* Rounded edges */
padding : 2px;
}
QSpinBox:disabled{
color: rgb(0, 0, 200);  /* Text color */
border: 1px solid rgb(255,255,255);  /* Border color */
border-radius: 1px;  /* Rounded edges */
padding : 2px;
}

QGroupBox{
color: rgb(0, 0, 150);  /* Text color */
font-weight: bold;  /* Bold text */
}

QLabel{
color: rgb(0, 0, 150);  /* Text color */
}

QPlainTextEdit{
background-color: rgb(255,255,255);
border-radius: 5px;  /* Rounded edges */
}

QLineEdit:endabled{
background-color: rgb(255,255,255);
color: rgb(0, 0, 150);  /* Text color */
border: 3px solid rgb(255,155,50);  /* Border color */
border-radius: 5px;  /* Rounded edges */
padding : 2px;
}

QLineEdit:disabled{
background-color: rgb(230,230,230);
color: rgb(0, 0, 150);  /* Text color */
border: 1px solid rgb(255,255,255);  /* Border color */
border-radius: 5px;  /* Rounded edges */
padding : 2px;
}

QComboBox{
background-color: rgb(220,220,220);
color: rgb(0, 0, 150);  /* Text color */
border: 3px solid rgb(255,155,50);  /* Border color */
border-radius: 5px;  /* Rounded edges */
padding : 2px;
}

QTabBar::tab {
    color: rgb(0, 0, 150); 
    border: 2px solid rgb(255,155,50);
    padding: 5px;
    border-top-left-radius: 5px;
    border-top-right-radius: 5px;
	min-width:80px;
}

QTabBar::tab:selected {
    color:rgb(0, 0, 150); 
    background-color: rgb(255, 255, 255); ;
    font-weight: bold;
	border: 3px solid rgb(235, 135, 30);
}

QTabBar::tab:hover {
    color: rgb(0, 0, 200); /* Slightly lighter orange */
	background-color: rgb(245, 245, 255); ;
}</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QGroupBox" name="rot_mot_groupbox">
     <property name="title">
      <string/>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Target Pos (deg):</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QDoubleSpinBox" name="target_pos_spinbox">
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
        <property name="buttonSymbols">
         <enum>QAbstractSpinBox::NoButtons</enum>
        </property>
        <property name="maximum">
         <double>360.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Current Pos (deg):</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QDoubleSpinBox" name="current_pos_spinbox">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
        <property name="buttonSymbols">
         <enum>QAbstractSpinBox::NoButtons</enum>
        </property>
        <property name="maximum">
         <double>1000.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Speed</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Acceleration</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QDoubleSpinBox" name="acc_spinbox">
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
        <property name="buttonSymbols">
         <enum>QAbstractSpinBox::NoButtons</enum>
        </property>
        <property name="minimum">
         <double>1.000000000000000</double>
        </property>
        <property name="maximum">
         <double>100.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QDoubleSpinBox" name="speed_spinbox">
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
        <property name="buttonSymbols">
         <enum>QAbstractSpinBox::NoButtons</enum>
        </property>
        <property name="minimum">
         <double>1.000000000000000</double>
        </property>
        <property name="maximum">
         <double>1000.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QPushButton" name="move_button">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Move</string>
        </property>
       </widget>
      </item>
      <item row="1" column="3">
       <widget class="QPushButton" name="move_to_home_button">
        <property name="text">
         <string>Move To Home</string>
        </property>
       </widget>
      </item>
      <item row="2" column="3">
       <widget class="QPushButton" name="set_home_button">
        <property name="text">
         <string>Set Home</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
